PyTorch/SpeechSynthesis/FastPitch/fastpitch/transformer.py

```diff
@@ -192,7 +192,7 @@ def __init__(self, n_layer, n_head, d_model, d_head, d_inner, kernel_size,
                     dropatt=dropatt, pre_lnorm=pre_lnorm)
             )
 
-    def forward(self, dec_inp, seq_lens=None, conditioning=0):
+    def forward(self, dec_inp, seq_lens=None, conditioning=0, conditioning_2=0): #here when called we add speaker or other discrete condition
```

Comment:
you could make condition a tuple, or rename the conditionings to conditioning_speaker conditioning_other